docs/configuration/upstream/tests.md

```diff
@@ -511,6 +511,37 @@ execute:
 
 Packit Service communicates with Testing Farm via its [API][testing-farm-api].
 
+### Public IDs of Packit tokens
+
+:::caution
+
+This feature has not been tested and verified by Packit Team.
```

Comment:
I would probably wait until https://github.com/packit/deployment/issues/564 is resolved